composer.json

```diff
@@ -49,7 +49,8 @@
         "filp/whoops": "^2.3",
         "phpunit/phpunit": "^8.1",
         "squizlabs/php_codesniffer": "^3.5",
-        "symfony/var-dumper": "^4.3"
+        "symfony/var-dumper": "^4.3",
+        "barryvdh/laravel-ide-helper": "^2.6"
```

Comment:
`b` should be somewhere at the start of the list

Reply:
✓ done